hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -457,6 +463,8 @@ def __repr__(self):
 
     @classmethod
     def all(cls) -> List["HealthCheck"]:
+        # Skipping of deprecated attributes is handled in HealthCheckMeta.__iter__
+        # TODO: note_deprecation() and write a codemod for HC.all() -> list(HC)
         return list(HealthCheck)
```

Comment:
Want to take this on as a follow-up PR?

Reply:
Sure! I can work on this after the cacheing :)